packages/framework/presence/src/latestValueManager.ts

```diff
@@ -327,12 +344,12 @@ export function latest<T extends object | null, Key extends string = string>(
 		>,
 	): {
 		initialData: { value: typeof value; allowableUpdateLatencyMs: number | undefined };
-		manager: InternalTypes.StateValue<LatestRaw<T>>;
+		manager: InternalTypes.StateValue<LatestRaw<T> & Latest<T>>;
```

Comment:
I'm confused why the intersection works. Is it because we know we only ever pass one or the other? I guess it's not a runtime problem unless there's some type-guarding going on based on properties in one or the other that are mutually exclsuive, but that's not the case here.

Reply:
> that is the original lie.

Ahhhh, yes. I see now.